Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -1614,6 +1614,26 @@ bool CClientVehicle::SetRotorSpeed(float fSpeed)
     }
 }
 
+bool CClientVehicle::SetWheelsRotation(float fRot1, float fRot2, float fRot3, float fRot4) noexcept
+{
+    if (!m_pVehicle)
+        return false;
+
+    switch (m_eVehicleType)
+    {
+        case CLIENTVEHICLE_CAR:
+        case CLIENTVEHICLE_MONSTERTRUCK:
+        case CLIENTVEHICLE_QUADBIKE:
+        case CLIENTVEHICLE_BIKE:
+        case CLIENTVEHICLE_BMX:
+        case CLIENTVEHICLE_TRAILER:
+            m_pVehicle->SetVehicleWheelRotation(fRot1, fRot2, fRot3, fRot4);
```

Comment:
I think you could make `CVehicle::SetVehicleWheelRotation` returning bool. It would be cleaner than having two switches in both methods.

Reply:
Done!